OMGVA PoS/Program.cs

```diff
@@ -105,6 +109,12 @@
 {
     app.UseSwagger();
     app.UseSwaggerUI();
+
+    using var scope = app.Services.CreateScope();
+    var dbContext = scope.ServiceProvider.GetRequiredService<OMGVADbContext>();
+    var logger = scope.ServiceProvider.GetRequiredService<ILogger<DbInitializer>>();
+    var dbInitializer = new DbInitializer(dbContext, logger);
+    await dbInitializer.InitDb(initDatabaseAction);
```

Comment:
You should wrap these lines in #IF DEBUG so that this code gets generated only when debug compilation is targeted. Same with dbInitializer whole class.

Reply:
It's already in `app.Environment.IsDevelopment()` condition, db initialization is for development purposes (for example, use mock data for testing), not just debugging

Reply:
I think I missed the app.Environment.IsDevelopment() condition which I believe I remember does the same. Also, for future reference, debug builds are usually used for development.